core/src/edu/asu/c3simulator/screens/MarketingWizard.java

```diff
@@ -0,0 +1,362 @@
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.CheckBox;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.List;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+import edu.asu.c3simulator.util.Product;
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * This is the second page of the marketing campaign wizard.
+ * It displays product/company information and marketing campaign options to enable the user to customize their campaign.
+ * 
+ * @author Alyahya, Mohammed
+ */
+public class MarketingWizard implements Screen
+{
+	private Game game;
+	private Stage stage;
+	private Skin skin;
+	private Product selectedProduct = null;
+	private String campaignMethod = "", marketingOption = "";
+	private MarketingScreen firstWizardPage;
+	private Table campaignOptions;
+	
+	public MarketingWizard(Game game, MarketingScreen previousWizardPage)
+	{
+		this.game = game;
+		this.stage = new Stage();
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		
+		firstWizardPage = previousWizardPage;
+		
+		selectedProduct = firstWizardPage.getSelectedProduct();
+		campaignMethod = firstWizardPage.getSelectedMarketingMethod();
+		
+		// TODO add Corner Advisor
+		// TODO add Home Button
+		
+		NavigationPanel navigationPanel = createNavigationPanel();
+		
+		Table mainTable = createMainTable();
+		mainTable.setPosition(
+				stage.getWidth() - 0.01f * stage.getWidth() - mainTable.getWidth(),
+				0.01f * stage.getHeight());
+		
+		stage.addActor(navigationPanel);
+		stage.addActor(mainTable);
+	}
+	
+	/**
+	 * The method creates the navigation panel.
+	 * 
+	 * @return the NavigationPanel actor that was created.
+	 */
+	private NavigationPanel createNavigationPanel()
+	{
+		NavigationPanel navigationPanel = new NavigationPanel(game, skin);
+		
+		// TODO Check if all screens should be left null since leaving the wizard should
+		// only be done through the back button.
+		navigationPanel.addButton("Products", null);
+		navigationPanel.addSubButton("Products", "Pre-Market", null);
+		navigationPanel.addSubButton("Products", "Current", null);
+		navigationPanel.addSubButton("Products", "Retired", null);
+		navigationPanel.addButton("Product Growth", null);
+		navigationPanel.addSubButton("Product Growth", "Supply", null);
+		navigationPanel.addSubButton("Product Growth", "Demand", null);
+		navigationPanel.addButton("Employment", null);
+		navigationPanel.addButton("Marketing", null);
```

Comment:
This pattern feels nostalgic. Is there any code we can reuse? Factory methods perhaps?